source/css/sass/patterns/molecules/teaser.scss

```diff
@@ -0,0 +1,30 @@
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.teaser-list {
```

Comment:
`.teaser-list` shouldn't nest `teaser` styles unless required to get the specificity (which in itself can be a sign that something's wrong somewhere). By keeping nesting to an absolute minimum (apart from pseudo elements and pseudo classes), we can avoid most specificity problems from biting us later.